### DISCUSSION TITLE
Ubuntu: Error - Cannot download packages whilst offline

### DISCUSSION THREAD

**Discussion by 1artist** [cockpit-project/cockpit#18523] [Q&A] [answered]
I am running Ubuntu Server 22.04.2 LTS, codename "Jammy".  I installed Cockpit and it was working great. Then I installed the Navigator application and I started getting this error when I refresh the applications screen:
"Error - Cannot download packages whilst offline"

I do not understand the error because the Network log says "carrier: link connected". So I am thinking my installing the Navigator application somehow messed up my Cockpit app.  Has anybody out there had any success installing Navigator into Cockpit? Because it never worked for me after using the two installations commands below:

```
$ wget https://github.com/45Drives/cockpit-navigator/releases/download/v0.5.10/cockpit-navigator_0.5.10-1focal_all.deb
# apt install ./cockpit-navigator_0.5.10-1focal_all.deb
```

In short, what is the correct way to perform a complete software purge of Cockpit and Navigator using terminal commands?  I figure it will be a lot easier to remove all of it and start over again with a new installation of Cockpit.   Any help will be greatly appreciated.

**Comment by martinpitt** [comment, marked as answer]
That error message is a well-known issue on Ubuntu between PackageKit and netplan. The navigator plugin is just accidental, I really doubt that it has any influence on it. I suppose what happened is that the automatic apt timer for refreshing package indexes fired, and the Software Updates page cannot load the information due to that.

Please see https://cockpit-project.org/faq.html#error-message-about-being-offline for the details why this happens, and how to work around that.
